theano/sandbox/cuda/tests/test_cusolver.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import absolute_import, division, print_function
+
+import unittest
+import numpy
+import theano
+
+from theano.tests import unittest_tools as utt
+
+# Skip tests if cuda_ndarray is not available.
+from nose.plugins.skip import SkipTest
+import theano.sandbox.cuda as cuda_ndarray
+from theano.misc.pycuda_init import pycuda_available
+from theano.sandbox.cuda.cusolver import cusolver_available
+
+from theano.sandbox.cuda import cusolver
+
+if not cuda_ndarray.cuda_available:
```

Comment:
The problem is still there, but now I understand why!

```
  File "/home/travis/build/Theano/Theano/theano/sandbox/cuda/tests/test_cusolver.py", line 13, in <module>
    from theano.sandbox.cuda.cusolver import cusolver_available
  File "/home/travis/build/Theano/Theano/theano/sandbox/cuda/cusolver.py", line 6, in <module>
    from theano.sandbox.cuda import CudaNdarray, GpuOp
ImportError: cannot import name CudaNdarray

CudaNdarray isn't availble when there is no nvcc and GPU. I'll comment at the right place for the import change needed.
```

This mostly mean we can't import the file cusolver for now.